esf.tf

```diff
@@ -85,7 +90,8 @@ module "esf-lambda-function" {
 }
 
 resource "aws_lambda_event_source_mapping" "esf-source-sqs-event-mapping" {
-  event_source_arn                   = aws_sqs_queue.esf-queue.arn
+  for_each = toset(data.aws_sqs_queue.esf-queue.arn)
```

Comment:
👍

Reply:
instead of `data.aws_sqs_queue.esf-queue` we can directly use `var.sqs_esf_names`

Reply:
Done